src/components/Requirements.vue

```diff
@@ -11,15 +11,17 @@
       </div>
         <!-- TODO change for multiple colleges -->
         <div v-if="index==2" class="major">
-          <div :style="{'border-bottom': major.display ? `2px solid #${reqGroupColorMap[req.group][0]}` : ''}"  @click="activate(id)" class="major-title" v-for="(major, id) in majors" :key="major.id">
+          <div :style="{'border-bottom': major.display ? `2px solid #${reqGroupColorMap[req.group][0]}` : ''}"
```

Comment:
Could you tag me in a monday item to get rid of this inline css? Mention it is at the top of Requirements.vue on this line and around 21-25? Thanks

Reply:
Ooop sorry bout dat. Got it